app/views/editor/cinematic/CinematicEditorComponent.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div class="container">
+    <div v-if="!cinematicSlug">
+      <div class="row">
+        <div class="col-md-12"><h1>{{ heading }}</h1></div>
+      </div>
+      <div class="row">
+        <!-- List of cinematics -->
+        <ul>
+          <editor-list
+            v-for="cinematic in cinematicList"
+            :key="cinematic.slug"
+            :text="cinematic.name"
+            :slug="cinematic.slug"
+            ></editor-list>
+            <li><button v-on:click="createCinematic">+</button></li>
+        </ul>
+        
+      </div>
+    </div>
+
+    <div v-else>
+      <!-- Have a cinematic Slug -->
+      <div class="row">
+        <div class="col-md-8"><h1>{{ heading }}</h1></div>
+        <div class="col-md-4">
+          <span>There is no autosave. Please click this button often.</span>
+          <button v-on:click="saveCinematic" :disabled="state.saving || !cinematic">save</button>
+          <button><a href="/cinematic">Play View (please save first)</a></button>
```

Comment:
Would this open `/cinematic` generally or with the slug of the selected cinematic?

Reply:
Only generally right now.

Reply:
Yeah. Punted as we'll need to migrate to the Vue router. Thanks for the suggestion though!